runners/flip

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd /ATO/context
+python3 /opt/flip/flip.py /ATO/code < /ATO/input
```

Comment:
Please follow the standard format for these runners, even if the language does not accept any arguments or options (yet).

```suggestion
/ATO/yargs %1 /ATO/options /ATO/yargs %2 /ATO/arguments python /opt/flip/flip.py %1 /ATO/code %2 < /ATO/input
```